Sources/View/ViewModelCollectionReusableView.swift

```diff
@@ -9,4 +9,11 @@ open class ViewModelCollectionReusableView<ViewModel>: CollectionReusableView, R
     }
 
     open func setUpBindings() {}
+
+    open override func prepareForReuse() {
+
+        viewModel = nil
```

Comment:
why do you set this to `nil` **before** calling `super`? 🤔

Reply:
My mistake while copying and pasting stuff around. Should be in the inverse order.